experiments/2multifsound.py

```diff
@@ -3,112 +3,49 @@
 # write an experiment that creates a new control program.
 import os
 import sys
+import copy
 
 BOREALISPATH = os.environ['BOREALISPATH']
-#sys.path.append(BOREALISPATH + "/experiment_prototype")
+sys.path.append(BOREALISPATH)
 
-#import test
 from experiment_prototype.experiment_prototype import ExperimentPrototype
+import experiments.superdarn_common_fields as scf
 
 class TwoMultifsound(ExperimentPrototype):
 
     def __init__(self):
         cpid = 350300
```

Comment:
Is this CPID correct?

Reply:
I dont think there is an assigned one for this mode.